lib/knackhq/client/recursive_symbolize_keys.rb

```diff
@@ -0,0 +1,21 @@
+# Recursively symbolizes keys in hash
```

Comment:
It doesn't seem clear to me where this functionality is coming from, the only hint in the client is the require statement.